src/controllers/product.controller.js

```diff
@@ -0,0 +1,10 @@
+const productService = require("../services/product.service");
+const { catchAsync } = require("../utils/error");
+
+const getNewProducts = catchAsync(async (req, res) => {
+  const orderBy = req.query.orderBy;
+  const limitNum = req.query.limitNum;
+  res.status(200).json(await productService.getNewProducts(orderBy, limitNum));
```

Comment:
```suggestion
  return res.status(200).json(await productService.getNewProducts(orderBy, limitNum));
```
응답 후 더이상 처리할 로직이 없다면 return 으로 함수를 마무리해주세요!